mlrun/feature_store/api.py

```diff
@@ -276,6 +276,11 @@ def get_online_feature_service(
     service.initialize()
 
     # todo: support remote service (using remote nuclio/mlrun function if run_config)
+    for old_name in service.vector.get_feature_aliases().keys():
```

Comment:
what is this change?
and if not related to the #todo, please move it above the #todo

Reply:
for support alias

Reply:
reverted - old bug that I found not relevant for this pr